scripts/install_dev_deps.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
 # Merlin, utop, ocp-indent, ocamlformat, and patdiff are all for developer assistance
-# Menhir is our parsing library and annoyingly its module name does not match
-# its library name, so we install it manually here.
-opam install -y core_kernel menhir ppx_deriving merlin utop ocp-indent patdiff ocamlformat
+opam install -y merlin utop ocp-indent patdiff ocamlformat
```

Comment:
I separated the deps for developers from the deps needed to build because patdiff has some dependency that wasn't building properly in the static environment.